dotnet/src/Connectors/Connectors.OpenAI/AzureSdk/OpenAIStreamingChatMessageContent.cs

```diff
@@ -30,6 +30,16 @@ public sealed class OpenAIStreamingChatMessageContent : StreamingChatMessageCont
     /// </summary>
     public string? FunctionArgument { get; set; }
 
+    /// <summary>
+    /// The ID of the tool call update.
+    /// </summary>
+    public string ToolCallUpdateId { get; set; }
+
+    /// <summary>
+    /// The reason why the completion finished.
+    /// </summary>
+    public CompletionsFinishReason? FinishReason { get; set; }
```

Comment:
This is going to conflict (logically) with https://github.com/microsoft/semantic-kernel/pull/4153. Can you handle any merging / updating that's necessary to whichever PR merges second?